### ISSUE TITLE
Procedural Sky in a WorldEnvironment (including the default default_env.tres) prevents scene from running on a Samsung Galaxy S4 (Android 5.0.1)

### ISSUE THREAD

**Issue by NanitesNanites** [godotengine/godot#55999]
### Godot version

3.4 stable

### System information

Samsung GT-I9505: Qualcomm Snapdragon 600 (APQ8064T), GPU: Qualcomm Adreno 320, Android 5.0.1

### Issue description

A 2D project will not run on a Galaxy S4 with Android 5.0.1 if there is either a WorldEnvironment node or the default_env.tres is set in Project Settings as default environment, if either of those contains a Sky Background with a Procedural Sky. Even a freshly created Procedural Sky with all default settings will cause the project to give a black screen and an Android Timeout ("Name isn't responding, Wait/Close") Message after a while.

Happens with both GLES2 and GLES3. Not sure if other phones are affected or different Android versions on this one, as I only have this one where I can reproduce it. Phone was set back to factory defaults just to make sure it's not some weird setting.

### Steps to reproduce

Basically a new empty 2D project will already fail to run if the default environment is included in the Project Settings.

### Minimal reproduction project

See above, just any new project will do.

**Comment by clayjohn** [comment]
We've seen variations of this same issue before, but usually they are restricted to GLES3 only.

Can you try the following to see if they work around the issue for you:
1. set [radiance_size](https://docs.godotengine.org/en/stable/classes/class_sky.html#class-sky-property-radiance-size) in your sky to 32
2. For GLES3 only reduce [irradiance_max_size](https://docs.godotengine.org/en/stable/classes/class_projectsettings.html#class-projectsettings-property-rendering-quality-reflections-irradiance-max-size) in the project settings
3. Ensure that [high_quality_ggx](https://docs.godotengine.org/en/stable/classes/class_projectsettings.html#class-projectsettings-property-rendering-quality-reflections-high-quality-ggx-mobile) is set to false in the project settings

**Comment by NanitesNanites** [comment]
Tried all 3 but none made a difference, both GLES2 and 3.

I set irradiance_max_size to 128, 64 and 32 (doesn't le me go lower).
Same black screen in all cases.

**Comment by NanitesNanites** [comment]
Probably the same issue as this: https://github.com/godotengine/godot/issues/26924

Should have searched better...

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/26924.
